dev.py

```
@@ -332,6 +338,11 @@ def cmd_build(self):
         if sanitize:
             install_args.append(f"-Csetup-args=-Db_sanitize={sanitize}")
 
+        if ci:
+            install_args.append(f"-Csetup-args=-Dci=true")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)